README.md

```diff
@@ -206,6 +206,13 @@ signIn.renderEl(
       console.log(res.user);
       res.session.setCookieAndRedirect('https://acme.com/app');
 
+      // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
```

Comment:
I'll change the comment to include the ```if``` blocks. Also will change 'redirect' to 'finish'. 

I'm already using 'target', so will keep it unless you prefer 'stepUp'. I don't have a strong opinion for either, I chose 'target' just because the response type is already 'STEP_UP'